_posts/2019-03-07-strimzi-0.11.0-release.md

```diff
@@ -0,0 +1,191 @@
+
+<!--more-->
+
+# JBOD storage support for Kafka brokers
+
+Other than supporting the ephemeral and persistent storage, now Strimzi supports JBOD (Just a Bunch of Disks) storage for Kafka brokers (but not for Zookeeper nodes).
+
+The JBOD data storage configuration provides a way for having multiple disks combined to create a single logical disk, or volume.
+JBOD is one approach to providing increased data storage for Kafka brokers.
+It can also improve performance.
+A JBOD configuration is described by one or more volumes, each of which can be either ephemeral or persistent.
+
+To use JBOD with Strimzi, the storage `type` must be set to the new `jbod` value.
+Then the `volumes` property allows you to describe the disks that make up your JBOD storage array or configuration.
+
+Here's an example snippet with a JBOD configuration.
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    ...
+    storage:
+      type: jbod
+      volumes:
+      - id: 0
+        type: persistent-claim
+        size: 100Gi
+        deleteClaim: false
+      - id: 1
+        type: persistent-claim
+        size: 100Gi
+        deleteClaim: false
+```
+
+The Cluster Operator will configure the `log.dirs` Kafka parameter accordingly, in order to store messages logs on the different volumes.
+
+# Improved configuration for off cluster access
+
+This improvement was mostly driven and contributed by the community!
+A lot of developers asked about having more flexibility and extensibility on the listeners configuration for exposing the Kafka cluster outside of the Kubernetes/OpenShift cluster.
+
+When configuring the `listeners.external` section of the `Kafka` resource, it's now possible to leverage a new `overrides` property for a more powerful configuration.
+
+For example, by default, using a `route` listener, the route hosts are automatically assigned by OpenShift.
+Now, you can override the assigned route hosts by specifying the requested hosts.
+
+Here's an example snippet for overriding route hosts.
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    ...
+    listeners:
+      external:
+        type: route
+          authentication:
+            type: tls
+          overrides:
+            bootstrap:
+              host: bootstrap.myrouter.com
+            brokers:
+            - broker: 0
+              host: broker-0.myrouter.com
+            - broker: 1
+              host: broker-1.myrouter.com
+            - broker: 2
+              host: broker-2.myrouter.com
+```
+
+Another powerful new addition is about configuring the node ports when the `nodeport` listener is used.
+By default, the port numbers used for the bootstrap and broker services are automatically assigned by OpenShift or Kubernetes.
+Now, you can override the assigned node ports by specifying the requested port numbers.
```

Comment:
```suggestion
Strimzi 0.11.0 lets you override the assigned node ports by specifying the requested port numbers.
```

Reply:
We don't agree which version we should mention here :-)

Reply:
I changed in "This new Strimzi release lets you ..."